crypto-ffi/bindings/js/CoreCrypto.ts

```diff
@@ -58,20 +58,10 @@ export enum Ciphersuite {
  * Configuration object for new conversations
  */
 export interface ConversationConfiguration {
-    /**
-     * List of client IDs with administrative permissions
-     * Note: This is currently unused
-     */
-    admins?: Uint8Array[];
     /**
      * Conversation ciphersuite
      */
     ciphersuite?: Ciphersuite;
-    /**
-     * Duration in seconds after which we will automatically force a self_update commit
-     * Note: This isn't currently implemented
-     */
-    keyRotationSpan?: number;
```

Comment:
We *will* need to implement this in the future, so we should keep it no matter what.

Reply:
I created another `CustomConfiguration` struct in addition to `ConversationConfiguration`. This struct will hold everything not natively handled by openmls ; curently `key_rotation_span` and `wire_format`.  
This struct will be used only where strictly required: joining from welcome or from external commit. That way, only the minimal amount of information is required on those methods without superfluous fields (i.e. ciphersuites).